squib.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'gio2',                  '~> 3.4'
   spec.add_runtime_dependency 'gobject-introspection', '~> 3.4'
   spec.add_runtime_dependency 'highline',              '2.0.3'
+  spec.add_runtime_dependency 'mechanize',             '~> 2.7'
```

Comment:
Mechanize might be overkill for us - I think we can download a file with Ruby's stdlib?